site/docs/components/splitter/examples.mdx

```diff
@@ -0,0 +1,105 @@
+---
+  title:
+    $ref: ./#/title
+  layout: DetailComponent
+  sidebar:
+    exclude: true
+  data:
+    $ref: ./#/data
+---
+
+<LivePreviewControls>
+
+The Salt Splitter leverages the popular open-source library [react-resizable-panels](https://react-resizable-panels.vercel.app/), whilst adding styling, introducing extra functionality and making adjustments to component and prop names to better align with the rest of the components present in our design system.
+
```

Comment:
```suggestion

```

Reply:
I would say we get rid of this line, as we tend to go straight into examples in all other docs so it feels a bit on it's own there

Reply:
Done!